package.json

```diff
@@ -25,7 +25,6 @@
     "connect",
     "json",
     "rpc",
-    "connect",
```

Comment:
Didn't notice I had a duplicate. Care to create a separate PR for this instead of doing it inside this one? I like to not have unrelated changes.